.github/workflows/update.yml

```diff
@@ -0,0 +1,52 @@
+name: Update flake inputs
+on:
+  schedule:
+    # Update every Sunday and Wednesday
+    - cron: "51 3 * * 0,3"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - uses: cachix/install-nix-action@v13
+        with:
+          install_url: https://github.com/numtide/nix-unstable-installer/releases/download/nix-2.4pre20210415_76980a1/install
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+      - name: Prepare flakebot branch
+        run: |
+          git config --local user.email "flakebot@users.noreply.github.com"
+          git config --local user.name "flakebot"
+          git checkout -B flake-update
+      - name: Update & commit lockfile
+        run: |
+          nix flake update --commit-lock-file
+          echo "commit_info<<EOF" >> $GITHUB_ENV
+          git log --format=%B -1 >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```

Comment:
```suggestion
          cat >> $GITHUB_ENV <<EOS
          commit_info<<EOF
          git log --format=%B -1
          EOS
```

Or alternatively, inlining with `echo -e 'commit_info<<EOF\ngit.. > $GITHUB_ENV`.

I think both of these are cleaner.

Reply:
Does this work for multiline output? If so then it should be fine.

Reply:
Alright, seems like it works fine. I'll commit these changes.

Reply:
I don't think we were looking to do multiline output anyway, IIUC. We just wanted to assign a value to a variable, correct? As explained [here](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable).

If not, then I might have misunderstood the purpose of your code.

Reply:
`git log --format=%B -1` will print the last commit's message, which will span multiple lines in this case. This is so that in the next step we can insert the commit message in the PR body, like you see [here](https://github.com/i077/system/pull/48).

Reply:
Ah OK, I get it now. Well, the syntax we switched to doesn't preserve newlines AFAICT. :/

Reply:
I tried it in the terminal and it works on my end: 
![image](https://user-images.githubusercontent.com/2789926/123663542-7e938d00-d804-11eb-94c4-9c6a37d586d3.png)

Reply:
Doesn't work on my system :confused: 

![20210628_18h59m23s_grim](https://user-images.githubusercontent.com/101816/123675435-3be8a980-d832-11eb-94c1-9cf19174e30c.png)

Reply:
```
[asymmetric@tachikoma:~/code/wasmer]$ bash --version
GNU bash, version 4.4.23(1)-release (x86_64-unknown-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

Reply:
Ah, I guess it works on zsh then. Shell scripting 🙄